libs/content/question-bank-ppl/content/020/020.02/020.02.01/020.02.01.01/questions.yaml

```diff
@@ -1,6 +1,108 @@
-- id: HP02002010102
+- id: HPL0003
+  relatedQuestions: []
+  externalIds: []
+  annexes: []
   learningObjectives: 
     - 020.02.01.01.02
-- id: HP02002010103
+  explanation: ""
+  variant:
+    type: simple
+    question:  What is the percentage of oxygen in the atmosphere at 6000 ft? (1,00 P.)
+    options: 
+    - id: HPL0003001
+      text: 21 %
+      correct: true
+      why: ""
+    - id: HPL0003002
```

Comment:
Future Work: If you have time / patience, you could reassign these ids. Answers IDs should be a `AXXXXXX` (an `A` followed by 7 uppercase characters + numbers), and Questions `QXXXXXX`.